packages/viz/__tests__/Layer.spec.ts

```diff
@@ -1,6 +1,6 @@
 import { Layer as DeckLayer, log } from '@deck.gl/core';
 import { Layer } from '../src/lib/Layer';
-import { defaultStyles } from '../src/lib/style';
+import { getStyles } from '../src/lib/style';
```

Comment:
Why not to keep `getDefaultStyles`?. I find it more clear

Reply:
Because it gets styles using the values of the user, so it's not just getting the defaults.

Maybe we can move it to another module

Reply:
Moved